File: Storyboard.Web/Models/Home/StoryOverviewSummary.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net;
using System.Net.Http;
using System.Web.Http;

namespace Storyboard.Web.Models.Home
{
    public class StoryOverviewSummary
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Synopsis { get; set; }
    }
}

File: Storyboard.Web/App/Home/Models/StoryOverviewSummary.ts
/// <reference path="../homemodule.ts" />
module Home {
    export interface IStoryOverviewSummary {
        Id : number
        Title : string
        Synopsis: string
    }
    export class StoryOverviewSummary implements IStoryOverviewSummary {
        Id: number
        Title: string
        Synopsis: string
    }
} 